automatic/es/es.nuspec

```
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>es</id>
+    <version>0.0.1</version>
+    <title>Everything Command Line</title>
+    <authors>David Carpenter</authors>
+    <owners>chocolatey-community</owners>
```

Since you are creating the new package, owners should include you own name or username as well.

```suggestion
    <owners>chocolatey-community,TheCakeIsNaOH</owners>
```

This is purely used for historical purposes though

---

Done